Create ui-c8y-1018-503-94-smart-rule-contains-unexpected-child-mo-6409-graft-release-y-2024.md (#2161)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-94-smart-rule-contains-unexpected-child-mo-6409-graft-release-y-2024.md

---
date: ""

> title: Smart Rule contains unexpected child MO (#6409) [GRAFT][release/y2024] (#6465)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Smart rules may contain unexpected child managed objects
```

> title: Smart Rule contains unexpected child MO (#6409) [GRAFT][release/y2024] (#6465)

Reviewer: ```suggestion
title: Smart rules list now contains a children column if child assets are assigned to a managed object
```

> title: Smart rules list now contains a children column if child assets are assigned to a managed object

Reviewer: ```suggestion
title: Smart rules list now always shows the children column properly if child assets are assigned to a managed object
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57897
version: 1018.503.94
---

> Smart Rule contains unexpected child MO (#6409) [GRAFT][release/y2024] (#6465)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, smart rules could contain unexpected child managed objects, which were not intended to be part of the rule. This could lead to unintended behavior or errors in the system. The issue has been resolved and smart rules now only contain the intended child managed objects as per their configuration. Existing smart rules with unexpected child managed objects will need to be reviewed and updated to ensure they work as intended.
```

> Smart Rule contains unexpected child MO (#6409) [GRAFT][release/y2024] (#6465)

Reviewer: ```suggestion
If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list,  a children column is displayed showing for which child device the smart rule is active.
```

> If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list,  a children column is displayed showing for which child device the smart rule is active.

Reviewer 2: ```suggestion
If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list, a children column is displayed showing for which child device the smart rule is active.
```

> If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list, a children column is displayed showing for which child device the smart rule is active.

Reviewer: ```suggestion
In the smart rules list of a group or a device, the children column, which shows for which child assets the smart rule is active, did not always show up properly. This issue has been fixed and the children column is now always displayed if the smart rule is activated for a child asset.
```
